projects/28_gltf/main.cpp

```diff
@@ -259,7 +259,8 @@ void ProjApp::LoadMaterial(
 
     grfx::GraphicsPipelineCreateInfo2 gpCreateInfo = {};
     gpCreateInfo.VS                                = {mVertexShader.Get(), "vsmain"};
-    gpCreateInfo.PS                                = {mPixelShader.Get(), "psmain"};
+    ppx::grfx::ShaderModule* psModule              = material.unlit ? mUnlitPixelShader.Get() : mPbrPixelShader.Get();
```

Comment:
Just to clarify, right now it wouldn't use PBR path, since the scene is hardcoded and the material is unlit in it?

Reply:
It would use both actually :)
The `material.unlit` depends on the GLTF file material definition. Only the skybox has the "unlit" marker, while the altimeter is classic PBR.
Each object is assigned 1 material, meaning we have 1 unlit material (skybox) and 1 'PBR' material ('PBR' just because this one is an ugly 'pbr' shader, not the real stuff)